Classes/Domain/Repository/GlossaryRepository.php

```diff
@@ -408,7 +410,7 @@ private function getAvailableLocalizations(int $pageId): array
 
     protected function getTargetLanguageIsoCode(Site $site, int $languageId): string
     {
-        // TODO add support for deprecated sys_language table
+        // @todo Add support for deprecated sys_language table.
```

Comment:
TODO can be removed, as v4 won't support sys_language anymore

Reply:
That was a sidechange/adjustment. Can we let it be as is ? We need to go through all classes anyway and I guess more cleanups will follow - or am I wrong ?

Reply:
Forget, will remove it.

Reply:
done.